docs/tethys_portal/configuration.rst

```diff
@@ -57,6 +57,7 @@ The following is a list of keys that can be added to the :file:`portal_config.ym
     * **ENABLE_RESTRICTED_APP_ACCESS**: app access can be restricted based on user object permissions when ``True``. Defaults to ``False``. If ``ENABLE_OPEN_PORTAL`` is set to ``True`` this setting has no effect. That is, users will have unrestricted access to apps independently of the value of this setting.
     * **TETHYS_WORKSPACES_ROOT**: location to which app workspaces will be synced when ``tethys manage collectworkspaces`` is executed. Gathering all workspaces to one location is recommended for production deployments to allow for easier updating and backing up of app data. Defaults to :file:`<TETHYS_HOME>/workspaces`.
     * **STATIC_ROOT**: the Django `STATIC_ROOT <https://docs.djangoproject.com/en/2.2/ref/settings/#static-root>`_ setting. Defaults to :file:`<TETHYS_HOME>/static`.
+    * **STATICFILES_USE_CDN**: uses a content delivery network (CDN) to serve JavaScript dependencies when ``True``. Defaults to ``True``. When set to ``False`` then you must run :cmd:`tethys gen vendor_js` to download the JS dependencies locally to be served by Tethys.
```

Comment:
It would make more sense to me if this was renamed so that you are “enabling” installing with NPM rather than disabling the default behavior? Perhaps something like `STATICFILES_USE_NPM = True`.

Reply:
Yes, using the CDN is the default. I've made the name change that you suggested, and toggled all of the boolean values.